packages/block-editor/src/components/block-list/content.scss

```diff
@@ -377,18 +377,24 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 	z-index: z-index("{core/image aligned left or right} .wp-block");
 }
 
-body.is-zoomed-out {
-	display: flex;
-	flex-direction: column;
+iframe[name="editor-canvas"].edit-site-visual-editor__editor-canvas.is-zoomed-out {
```

Comment:
Should we style `edit-site*` classes fromthe block editor package?

Reply:
mmh, good question. We were applying the class to the body within the iframe, but we are styling what's inside here too...

Reply:
I came here to say the same. It would be to also support post editor

Reply:
I think we should not. Ideally the block editor should not have any visual opinion on how zoomed out background looks. It does the scaling. Then the final implementation deals with styling this by consuming the state and learning it's in zoomed out mode and applying classes to its containers.

Reply:
I think it's fine to have opinionated styles if they make sense in all block editors in zoom out mode but yeah we shouldn't target edit-site here

Reply:
>  if they make sense in all block editors

How can we know :)

Reply:
> How can we know :)

Using good sense.

One way to try is to enable zoom-out setting in the playground story for instance and see if it looks correct.

Reply:
what's the playground story?

Reply:
I think

https://wordpress.github.io/gutenberg/?path=/docs/playground-block-editor--docs